### ISSUE TITLE
Feature Request: Provide official docker images

### ISSUE THREAD

**Issue by tsawada** [codecov/uploader#608]
Please provide official docker images for the uploader.

My use case is to run the uploader on Google Cloud Build. On Cloud Build, you need to include any executable within docker images to run. It'd be really convenient if I can just refer to `codecov/uploader` image, instead of building and maintaining one by myself.

Please let me know if you need any more information to clarify. Thanks in advance!

**Comment by cbix** [comment]
I agree an officially maintained Docker image with the latest uploader binary would be the ideal way to upload reports in most CI pipelines.
